torch/_inductor/codegen/cpp.py

```diff
@@ -620,6 +642,60 @@ def codegen_define_and_call(self, wrapper):
             "{}({})".format(kernel_name, ", ".join(call_args)),
         )
 
+    def get_kernel_path(self, code):
+        # TODO: this duplicates with CodeCache logic
+        ext = "so"
+        extra = cpp_compile_command("i", "o")
+        # \n is required to match with the CodeCache behavior
+        source_code = "\n" + code.getvalue()
+        basename = code_hash(source_code + extra)
+        subdir = os.path.join(cache_dir(), basename[1:3])
+        kernel_path = os.path.join(subdir, f"{basename}.{ext}")
+        return kernel_path
+
+    def cpp_codegen_define_and_call(self, wrapper):
```

Comment:
I guess the "define" part is common between python wrapper and cpp wrapper and can be shared while "call" part is different?